internal/security/bootstrapper/config/config.go

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright 2021 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+
+package config
+
+import (
+	bootstrapConfig "github.com/edgexfoundry/go-mod-bootstrap/config"
+)
+
+type ConfigurationStruct struct {
+	Writable  WritableInfo
+	StageGate StageGateInfo
+}
+
+type WritableInfo struct {
+	LogLevel        string
+	InsecureSecrets bootstrapConfig.InsecureSecrets
```

Comment:
Why is this needed? GetInsecureSecrets() below returns nil?

Reply:
i just copied from all similar other place to make it standardized.  And yes, it returns nil in function below.

Reply:
It isn't needed here. Please remove.

Reply:
removed